src/Bundle/test/src/BoardGameBlog/Infrastructure/Sylius/Grid/BoardGameGrid.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace App\BoardGameBlog\Infrastructure\Sylius\Grid;
+
+use App\BoardGameBlog\Domain\Model\BoardGame;
+use Sylius\Bundle\GridBundle\Builder\Action\CreateAction;
+use Sylius\Bundle\GridBundle\Builder\Action\DeleteAction;
+use Sylius\Bundle\GridBundle\Builder\Action\ShowAction;
+use Sylius\Bundle\GridBundle\Builder\Action\UpdateAction;
+use Sylius\Bundle\GridBundle\Builder\ActionGroup\ItemActionGroup;
+use Sylius\Bundle\GridBundle\Builder\ActionGroup\MainActionGroup;
+use Sylius\Bundle\GridBundle\Builder\Field\StringField;
+use Sylius\Bundle\GridBundle\Builder\GridBuilderInterface;
+use Sylius\Bundle\GridBundle\Grid\AbstractGrid;
+use Sylius\Bundle\GridBundle\Grid\ResourceAwareGridInterface;
+
+final class BoardGameGrid extends AbstractGrid implements ResourceAwareGridInterface
+{
+    public static function getName(): string
+    {
+        return 'app_board_game';
+    }
+
+    public function buildGrid(GridBuilderInterface $gridBuilder): void
+    {
+        $gridBuilder
+            ->orderBy('name', 'asc')
+            ->addField(
+                StringField::create('name')
+                    ->setPath('name.value')
+                    ->setLabel('Name')
+                    ->setSortable(true, 'name.value'),
+            )
+            ->addActionGroup(
+                MainActionGroup::create(
+                    CreateAction::create(),
+                ),
+            )
+            ->addActionGroup(
+                ItemActionGroup::create(
+                    ShowAction::create(),
+                    UpdateAction::create(),
+                    DeleteAction::create(),
+                ),
+            )
+        ;
+    }
+
+    public function getResourceClass(): string
+    {
+        return BoardGame::class;
```

Comment:
@lchrusciel In a future Grid bundle, we'll need to use the Resource instead of the Doctrine model.